src/rpc/core_rpc_server.h

```diff
@@ -59,6 +59,9 @@ namespace cryptonote
   class core_rpc_server: public epee::http_server_impl_base<core_rpc_server>
   {
   public:
+    static constexpr size_t THREADS = 32;
+    static constexpr size_t MAX_LONG_POLL_CONNECTIONS = 30;
```

Comment:
I think that this should be a configuration option: `--rpc-long-poll-connections` perhaps.  I think it's highly likely that public remote nodes will want to tweak this.

Reply:
I don't like that we have to start all these threads per connection, but given the current RPC implementation I don't think there's a nicer way to deal with it until we replace it with a proper bidirectional asynchronous persistent interface.